docs/api/utils/proxyWithComputed.mdx

```diff
@@ -7,36 +7,24 @@ description: ''
 
 # proxyWithComputed
 
-#### add a computation to a proxy
+#### add cached computation to a proxy
```

Comment:
I know `proxyWithComputed` doesn't _have_ to be used for caching, but that seems like the primary use case, so snuck this in...happy to drop.

Reply:
For a historical reason, we show the usage with proxy-memoize.
But, the primary use case is without proxy-memoize, so no caching.
proxy-memoize should be used if caching improves performance (which can be rare).

Reply:
Hm, thanks for pointing this out, b/c I'd assumed this caching pattern was the primary reason to use `proxyWithComputed` over object getters--but if that's not the case, what actually is the primary use case?

And is that a use case that cannot be served by object getters/setters? I know from an implementation perspective, the object getter will be invoked with `this` as the proxy, where as `proxyWithComputed` will have `get` invoked with a snapshot, but I'm struggling to guess a use case / situation where that would matter.